kinder/cmd/kinder/build/nodevariant/nodevariant.go

```diff
@@ -88,6 +89,11 @@ func NewCommand() *cobra.Command {
 		"",
 		"override the kubeadm binary existing in the image with the given version/build-label/file or folder containing the kubelet binary",
 	)
+	cmd.Flags().StringSliceVar(
+		&flags.File, "with-file",
+		nil,
+		"sourcePath:destPath pairs; copies file at sourcePath on the host to destPath inside the image",
```

Comment:
should we support generic copy of paths instead of files only and potentially rename this flag to `with-path`?

/cc @fabriziopandini

Reply:
That's an interesting point

Copy paths is more powerful, but as of today the use case for this flag are add a CNI manifest, add a contanerd config file, might be add a static pod patch.

So, unless @neolit123 have strong preferences, I leave to @prasadkatti to evaluate if we can switch to copy path with a reasonable effort within the context of this PR

Reply:
ok, leaving the decision to @prasadkatti

Reply:
I am fairly close to getting `with-file` to work. But I can evaluate how much change is required to get `with-path` to work. If it is not too big of an effort, I can do it just as part of this PR.

Reply:
Updated the PR to add `with-path` instead of `with-file`.